src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -81,28 +86,136 @@ void VulnerabilityScannerFacade::start(
         auto scanOrchestrator = std::make_shared<ScanOrchestrator>(
             m_indexerConnector, m_databaseFeedManager, m_reportSocketClient, m_internalMutex);
 
+        // Queues to process skipped events
+        m_deltaQueue = std::make_shared<Utils::SafeQueue<std::string, RocksDBQueue<std::string>>>(
+            RocksDBQueue<std::string>(DELTA_EVENTS_QUEUE_PATH));
+        m_syncQueue = std::make_shared<Utils::SafeQueue<std::string, RocksDBQueue<std::string>>>(
+            RocksDBQueue<std::string>(SYNCHRONIZATION_EVENTS_QUEUE_PATH));
+
+        // Threads to retry skipped events
+        m_deltaThread = std::thread(
+            [scanOrchestrator, this]()
+            {
+                while (!m_stopRetryThread.load())
+                {
+                    for (int i = 0; i < SLEEP_RETRY_THREADS_SEC; ++i)
```

Comment:
The 1 minute delay between processed events was a requirement. Probably 1 minute is too long and 30s could be better. Please @sebasfalcone validate this.

Reply:
After the call today, it's clear that we need to wait before processing the events because without the delay they would be lost without the opportunity of the OS sync. 
The idea is to have like a "burst".

Reply:
I understood now, yeah not practical one event per minute.